packages/cozy-mespapiers-lib/src/components/Search/search.js

```diff
@@ -28,8 +28,8 @@ export const index = new Document({
       'metadata:DObtentionDate', // io.cozy.files
       'metadata:passportNumber', // io.cozy.files
       'metadata:vinNumber', // io.cozy.files
-      'flexsearchProps:translatedQualificationLabel', // io.cozy.files
       'fullname', // io.cozy.contacts
+      'name', // io.cozy.files, io.cozy.contacts
```

Comment:
I think using the attributes order in the index declaration might be worth an explanation in comment/commit description ;)
But I feel like this is quite fragile, hopefully there is a better way to achieve this with flexsearch

Reply:
en effet, je rajoute un commentaire